docs/maps/reverse-geocoding-tutorial.asciidoc

```diff
@@ -162,10 +162,10 @@ Now that our web traffic contains CSA region identifiers, you'll visualize CSA r
 . Click *Choropleth*.
 . For *Boundaries source*: 
 .. Select *Points, lines, and polygons from Elasticsearch*.
-.. Set *Index pattern* to *csa*.
+.. Set *Data view * to *csa*.
```

Comment:
The asterisk after Data view shows up in the doc. Removing the space should fix it.
```suggestion
.. Set *Data view* to *csa*.
```